datasets/google_dei/diversity_annual_report/pipeline.yaml

```diff
@@ -0,0 +1,534 @@
+            type: "integer"
+            mode: "required"
+          - name: "gender_us"
+            description: "Gender of Googlers in the U.S."
+            type: "string"
+            mode: "required"
+          - name: "race_asian"
+            description: "The percentage of Googlers in the U.S. who identify as Asian"
+            type: "float"
+            mode: "nullable"
+          - name: "race_black"
+            description: "The percentage of Googlers in the U.S. who identify as Black"
+            type: "float"
+            mode: "nullable"
+          - name: "race_hispanic_latinx"
+            description: "The percentage of Googlers in the U.S. who identify as Hispanic or Latinx"
+            type: "float"
+            mode: "nullable"
+          - name: "race_native_american"
+            description: "The percentage of Googlers in the U.S. who identify as Native American"
+            type: "float"
+            mode: "nullable"
+          - name: "race_white"
+            description: "The percentage of Googlers in the U.S. who identify as White"
+            type: "float"
+            mode: "nullable"
+
+    - operator: "GoogleCloudStorageToBigQueryOperator"
+      description: "Task to load CSV data to a BigQuery table"
+      args:
+        task_id: "load_non_intersectional_representation_to_bq"
+        bucket: "{{ var.json.google_dei.storage_bucket }}"
+        source_objects: ["DAR/non_intersectional_representation.csv"]
+        source_format: "CSV"
+        destination_project_dataset_table: "google_dei.dar_non_intersectional_representation"
+        skip_leading_rows: 1
+        write_disposition: "WRITE_TRUNCATE"
+        schema_fields:
+          - name: "workforce"
+            description: "Overall and sub-categories"
+            type: "string"
+            mode: "required"
+          - name: "report_year"
+            description: "The year the report was published"
+            type: "integer"
+            mode: "required"
+          - name: "race_asian"
+            description: "The percentage of Googlers in the U.S. who identify as Asian"
+            type: "float"
+            mode: "nullable"
+          - name: "race_black"
+            description: "The percentage of Googlers in the U.S. who identify as Black"
+            type: "float"
+            mode: "nullable"
+          - name: "race_hispanic_latinx"
+            description: "The percentage of Googlers in the U.S. who identify as Hispanic or Latinx"
+            type: "float"
+            mode: "nullable"
+          - name: "race_native_american"
+            description: "The percentage of Googlers in the U.S. who identify as Native American"
+            type: "float"
+            mode: "nullable"
+          - name: "race_white"
+            description: "The percentage of Googlers in the U.S. who identify as White"
+            type: "float"
+            mode: "nullable"
+          - name: "gender_us_female"
```

Comment:
For the next few pertaining to gender, these columns, if they have "male" and "female" as the values actually are referring to biological sex (gender would be man, woman, non-binary, etc.) ([reference](https://www.hrc.org/resources/glossary-of-terms) - in this glossary, the two terms to refer to are "sex assigned at birth" and "gender identity")

I'd love to see the source data here and see if it got this wrong and if we're then perpetuating that (we may want to add a note about it) or if we got it wrong here and need to make a fix. I'm concerned that we don't have anything beyond male/female in this data especially wrt data ethics of anyone using this

Reply:
See later comment that this will be escalated internally